CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Update the `forbidden-io-function-checker` to check functions from imported modules as well as methods (according to their qualified name)
 - Update the `forbidden-io-function-checker` to flag aliases of forbidden functions
 - Update how error messages are overridden such that section headers are no longer required within the config file
-- Added `presistent_server` which recives the watch property changes through websockets and updates the HTML report
+- Added `presistent_server` which recives the watch property changes through websockets and updates the HTML repor
+- Added test case to `test_check.py` for better coverage of `colour_messages_by_type`
```

Comment:
This entry should be added to the "internal changes" section below; "Enhancements" and the other subsections are reserved for user-facing changes.